pkgs/development/compilers/ocaml/camlboot.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, enableParallelBuilding ? true
```

Comment:
Why do we need to disable this?

Reply:
I just figured it was good practice to offer the option to disable parallel building, I can remove the argument.

Reply:
You can set cores to 1 in nix.conf to disable parallel building. Please remove this.